lib/stdlib/src/supervisor.erl

```diff
@@ -514,9 +509,9 @@ handle_call(count_children, _From,  #state{dynamic_restarts = Restarts,
 handle_call(count_children, _From, State) ->
     %% Specs and children are together on the children list...
     {Specs, Active, Supers, Workers} =
-	lists:foldl(fun(Child, Counts) ->
-			   count_child(Child, Counts)
-		   end, {0,0,0,0}, State#state.children),
+	ch_fold(fun(_Id, Child, Counts) ->
+                        count_child(Child, Counts)
+                end, {0,0,0,0}, State#state.children),
```

Comment:
ch_foldl -> children_fold 
I think it is not unbearable long and clearer.

Reply:
Will do!